MojioSDK/Models/Core/Activity.swift

```diff
@@ -523,7 +528,21 @@ public struct RootActivity: RootActivityModel {
             self.target = nil
             self.location = try container.decodeIfPresentIgnoringCase(ActivityLocation.self, forKey: CodingKeys.location)
         }
-    }
+        
+        if let driverScore = try container.decodeIfPresentIgnoringCase(Float.self, forKey: CodingKeys.driverScore) {
```

Comment:
@andriy-morgunov-epam would be better to use `flatMap` as for `published` field. It will look a little bit shorter.

Reply:
@

> @andriy-morgunov-epam would be better to use `flatMap` as for `published` field. It will look a little bit shorter.

@andriimakukha Nice! Thanks!